info/description.md

```diff
@@ -68,7 +67,7 @@ from battle import ROLE
 
 - `unit` - Mobile fighting items. Appears from crafts. `ROLE.UNIT`
 - `tower` - Stationary fighting items. `ROLE.TOWER`
-- `center` - Command Centers. The main building. If it's destroyed, then a battle is over. `ROLE.CENTER`
+- `center` - Command Centers. The ain building. If it's destroyed, then the battle is over. `ROLE.CENTER`

```

Comment:
I've always been taught that there generally shouldn't be a comma before the word "and" unless your English professor deems otherwise.